cmd/endpoint.go

```diff
@@ -301,8 +284,8 @@ func (endpoints Endpoints) UpdateIsLocal() error {
 					endpoints[i].Hostname(),
 				)
 
-				if orchestrated && endpoints.doAllHostsResolveToLocalhost() {
-					err := errors.New("hosts resolve to same IP, DNS not updated on k8s")
+				if orchestrated && endpoints.doAnyHostsResolveToLocalhost() {
+					err := errors.New("hosts resolve 127.*, DNS not updated on k8s")
```

Comment:
Should we remove k8s from the error message?

Reply:
It is on purpose since it is meant for k8s here @krisis